CMakeLists.txt

```diff
@@ -170,14 +174,30 @@ set(GAMMARAY_BUILD_UI_DEFAULT ON)
 if(QNXNTO OR ANDROID)
   set(GAMMARAY_BUILD_UI_DEFAULT OFF)
 endif()
-gammaray_option(GAMMARAY_BUILD_UI "Build the GammaRay client and in-process UI." ${GAMMARAY_BUILD_UI_DEFAULT})
 
 gammaray_option(
   GAMMARAY_PROBE_ONLY_BUILD
   "Build only an additional probe configuration for an already existing launcher."
   OFF
 )
 
+gammaray_option(
+    GAMMARAY_CLIENT_ONLY_BUILD
+    "Build the client part only."
+    OFF
+)
+
+if (GAMMARAY_CLIENT_ONLY_BUILD)
+    set(GAMMARAY_BUILD_UI_DEFAULT ON)
+    set(BUILD_TESTING OFF)
```

Comment:
Why disable testing in this case?

Reply:
Currently they caused lots of build issues in those, I will have a closer looks now that there is no much comments for the review.

Reply:
Fixed